lib/mixins/property-effects.html

```diff
@@ -86,7 +89,7 @@
    * Runs all effects of a given type for the given set of property changes
    * on an instance.
    *
-   * @param {Object} inst The instance with effects to run
+   * @param {PropertyEffectsInstance | Polymer_PropertyEffects} inst The instance with effects to run
```

Comment:
Actually, PropertyEffectsInstance already extends Polymer_PropertyEffects so I can probably remove the type union here.

In general though, I would have to make a new type to do a union like you would want.